setup.py

```diff
@@ -25,7 +25,12 @@ def package_files(directory):
     url="https://github.com/equinor/semeio",
     description="Jobs and workflow jobs for Ert.",
     packages=find_packages(include=["semeio*"], exclude=["semeio.jobs.scripts"]),
-    entry_points={"ert": ["semeio_jobs = semeio.hook_implementations.jobs"]},
+    entry_points={
+        "ert": [
+            "semeio_jobs = semeio.hook_implementations.jobs",
+            "CorrelatedObsScaling = semeio.jobs.scripts.correlated_observations_scaling",  # noqa
```

Comment:
The comment # noqa is new for me, cool to have this option to the linter ignore any warnings in the line code. 
We do not have it in the #156, #157 and #159. Is it because we will have merged the 4 comments later?